src/main/java/mate/academy/service/MovieServiceImpl.java

```
@@ -0,0 +1,25 @@
+package mate.academy.service;
+
+import java.util.Optional;
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        //return movieDao.get(id).get();
+        Optional<Movie> optionalMovie = movieDao.get(id);
+        return optionalMovie.orElse(new Movie());
```

let's throw an exception if the optional is empty